testapi.pm

```diff
@@ -565,6 +565,7 @@ sub wait_screen_change(&@) {
     $args{similarity_level} //= 50;
 
     bmwqemu::log_call(timeout => $timeout, %args);
+    $timeout = bmwqemu::scale_timeout(timeout);
```

Comment:
So here you want to log the unscaled timeout and in `wait_idle` the scaled timeout? Doesn't look consistent to me. What does the rest of the code-base? And the parameter passed to the `scale_timeout` function misses a `$` sign.

Reply:
No, maybe my English or my sentence is not so good but I said the contrary :) I changed the code in `wait_idle` to log the unscaled timeout. I just double checked my commit and it's doing this.

Reply:
Right, sorry. I misunderstood the usage of REAL. But the `$` sign is still missing.

Reply:
Yes sorry! I resolved the conversation before pushing my fix...